packages/iov-bns/src/encode.ts

```diff
@@ -182,9 +191,13 @@ function buildSwapOfferTx(tx: SwapOfferTransaction): codecImpl.app.ITx {
 }
 
 function buildSwapCounterTx(tx: SwapCounterTransaction): codecImpl.app.ITx {
+  const creatorIdentity: PublicIdentity = {
```

Comment:
Yeah, this looks like the same destructuring again and again, I like my helper function idea....